src/components/Folder/Folder.tsx

```diff
@@ -273,23 +322,38 @@ const Folder = <T extends Conversation | Prompt>({
     if (searchTerm) {
       dispatch(UIActions.openFolder({ id: currentFolder.id }));
     }
-  }, [searchTerm]);
+  }, [currentFolder.id, dispatch, searchTerm]);
 
   useOutsideAlerter(dragDropElement, setIsSelected);
 
-  const draggingColor =
-    highlightColor === 'green' ? 'bg-green/15' : 'bg-violet/15';
-  const hoverIconColor =
-    highlightColor === 'green' ? 'hover:text-green' : 'hover:text-violet';
-  const textColor = highlightColor === 'green' ? 'text-green' : 'text-violet';
-  const bgColor = highlightColor === 'green' ? 'bg-green/15' : 'bg-violet/15';
+  const hoverIconColor = classNames(
+    highlightColor === 'green'
+      ? 'hover:text-green'
+      : highlightColor === 'violet'
+      ? 'hover:text-violet'
+      : 'hover:text-blue-500',
+  );
+  const textColor = classNames(
+    highlightColor === 'green'
+      ? 'text-green'
+      : highlightColor === 'violet'
+      ? 'text-violet'
+      : 'text-blue-500',
+  );
+  const bgColor = classNames(
+    highlightColor === 'green'
+      ? 'bg-green/15'
+      : highlightColor === 'violet'
+      ? 'bg-violet/15'
+      : 'bg-blue-500/20',
+  );
```

Comment:
Actually I'm not sure that this is working, because tailwind is checking for colors by some pattern and if we will not use them directly I'm not sure that this will work
Moreover we have some linters and classnames needed for them correctly working - they show is class exists or not

Reply:
@mikitabut, this method will work correctly as with `classNames`